api/wasm-interpreter/Cargo.toml

```diff
@@ -20,6 +20,8 @@ highlight = ["slint-interpreter/highlight"]
 
 [dependencies]
 slint-interpreter = { path = "../../internal/interpreter", default-features = false, features = ["std", "backend-winit", "renderer-winit-femtovg", "compat-1-0"] }
+slint = { path = "../../api/rs/slint", default-features = false, features = ["compat-1-0"] }
```

Comment:
Why?
There is also `slint_interpreter::invoke_from_event_loop`, that doesn't work?

Reply:
Ah yes, you're right. I somehow overlooked that.